libclamav/dsig.c

```diff
@@ -230,23 +252,33 @@ char *cli_getdsig(const char *host, const char *user, const unsigned char *data,
 
 int cli_versig(const char *md5, const char *dsig)
```

Comment:
While you're here, could I trouble you to change the return type and the `ret` variable to `cl_error_t`?  I've been trying to clean this up throughout the codebase whenever we encounter it.

Reply:
updated. Need some sleep now.

Reply:
Sleep well.  Let me know when you're done pushing up changes to the PR and I'll re-test, re-review.  If you have any idea what's causing freshclam to hang for this build, or if it doesn't hang for you - please let me know.